odbc/src/tests.rs

```diff
@@ -1,10 +1,14 @@
 use crate::{
+    errors::ODBCError,
     handles::{Connection, ConnectionState, Env, EnvState, MongoHandle, Statement, StatementState},
-    SQLAllocHandle, SQLFreeHandle,
+    SQLAllocHandle, SQLFreeHandle, SQLGetDiagRecW,
 };
 use odbc_sys::{Handle, HandleType, SqlReturn};
 use std::sync::RwLock;
 
+const ERROR_MESSAGE: &str = "[MongoDB][API] The feature SQLDrivers is not implemented\0";
+const UNIMPLEMENTED_FUNC: &str = "HYC00\0";
```

Comment:
it seems like these should both be local vars (or just inlined) in the `validate_diag_rec` helper function instead of being globals here